editor/src/components/canvas/ui-jsx-canvas.tsx

```diff
@@ -457,7 +458,7 @@ function useGetStoryboardRoot(
   StoryboardRootComponent: ComponentRendererComponent | undefined
   storyboardRootElementPath: InstancePath
   rootValidPaths: Array<InstancePath>
-  rootInstancePath: TemplatePath
+  rootInstancePath: InstancePath
```

Comment:
```suggestion
  rootInstancePath: TemplatePath
```

Please do not undo my work :sweat_smile: